test/fortune/multitenant-integration.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+var request = require('supertest');
+var should = require('should');
+var RSVP = require('rsvp');
+var _ = require('lodash');
+
+module.exports = function(options){
+  describe('fortune-multitenant integration', function(){
+    var ids, baseUrl;
+    beforeEach(function(){
+      ids = options.ids;
+      baseUrl = options.baseUrl;
+    });
+    it('should allow to create duplicate PK for different tenants', function(done){
+      request(baseUrl).post('/people')
+        .set('content-type', 'application/json')
+        .send(JSON.stringify({people: [
+          {email: 'dilbert@mailbert.com', _tenantId: 'testing'}
+        ]}))
+        .end(function(err, res){
+          should.not.exist(err);
+          res.statusCode.should.equal(201);
+          request(baseUrl).get('/people')
+            .end(function(err, res){
+              should.not.exist(err);
+              var body = JSON.parse(res.text);
+              var matches = _.filter(body.people, function(p){ return p.email === 'dilbert@mailbert.com'});
+              matches.length.should.equal(2);
+              (matches[0]._tenantId === matches[1]._tenantId).should.not.be.ok;
+              done();
+            });
+        });
+    });
+    it('should not let create duplicate business PK within single tenant', function(done){
+      request(baseUrl).post('/people')
+        .set('content-type', 'application/json')
+        .send(JSON.stringify({people: [
+          {email: 'dilbert@mailbert.com', _tenantId: 'testing'}
+        ]}))
+        .expect(201)
+        .end(function(err){
+          should.not.exist(err);
+          request(baseUrl).post('/people')
+            .set('content-type', 'application/json')
+            .send(JSON.stringify({people: [
+              {email: 'dilbert@mailbert.com', _tenantId: 'testing'}
+            ]}))
+            .expect(500)
```

Comment:
Yep, exactly. And we have no way to figure out if it was duplicate primary key or connection failure atm. https://github.com/flyvictor/fortune/issues/150